client/src/components/Intro/intro.test.tsx

```diff
@@ -1,55 +1,28 @@
 import React from 'react';
```

Comment:
changed this test to use @testing-library/react which resolves all linting errors